src/modules/hardware-wallet/src/LedgerApp.ts

```diff
@@ -0,0 +1,218 @@
+import { ReturnCode, Instruction, CLA, AppState } from './types'
```

Comment:
Comment on the filename - this is really a LederNanoSApp, isn't it? The Ledger Nano X app would have to be different?

Reply:
This interface will be the same for all ledger devices